app/views/projects/show.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1>Project Name: <%=@project.name%></h1>
+<h3>Project Material: <%=@project.material%></h3>
+
+<h3>Project Challenge: <%=@project.challenge.theme%></h3>
+<p> Number of participants: <%=@project.contestants.count %>
```

Comment:
Even though it's a simple calculation, `.count` is still a calculation so by strict MVC standards, this should live in the model. You can encapsulate this into a model method (like you did for average experience) and then test it in a model spec

Reply:
This is just one minor MVC infraction -- and the rubric allows for 1 infraction!